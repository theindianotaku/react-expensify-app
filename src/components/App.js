import React, { Component } from 'react';


class App extends Component {
  render() {
    return (
      <div>
        <p>Hello there! Expensify here.</p>
      </div>
    );
  }
}

export default App;
