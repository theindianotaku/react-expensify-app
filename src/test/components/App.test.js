import React from 'react';
import { shallow } from 'enzyme';
import App from '../../components/App';

test('should render App component', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
