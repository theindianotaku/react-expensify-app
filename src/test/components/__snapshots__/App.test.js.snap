// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(Component) />
</div>
`;
